test/membrane/rtp/session_bin_test.exs

```diff
@@ -28,6 +32,46 @@ defmodule Membrane.RTP.Session.ReceiveBinTest do
 
   @fmt_mapping %{96 => {:H264, 90_000}, 127 => {:MPA, 90_000}}
 
+  @spec inspect_packet_fields(Packet.t(), %{}) :: boolean()
+  defp inspect_packet_fields(packet, fields_values), do: compare(packet, fields_values)
+
+  @spec compare(any(), any()) :: boolean()
+  defp compare(left, right) when is_map(left) or is_struct(left) do
+    if is_map(right) do
+      right
+      |> Enum.all?(fn {k, v} ->
+        case Map.has_key?(left, k) do
+          true ->
+            compare(Map.get(left, k), v)
+
+          _ ->
+            false
+        end
+      end)
+    else
+      false
+    end
+  end
+
+  defp compare(left, right), do: left == right
+
+  # asserts all specified buffers were received by the sink
+  defp assert_specified_buffers(_pipeline, _sink, []), do: :ok
+
+  defp assert_specified_buffers(pipeline, sink, field_specyfications) do
+    assert_sink_buffer(pipeline, sink, buffer, 10000)
```

Comment:
why such a big timeout?

Reply:
Yeah, it can be removed from here, but anyway there is still a problem that sometimes we have to wait quite a few seconds until all RR are collected (as I observed usually SR-s are delivered almost immediately).